src/tranche-space.rain

```diff
@@ -390,12 +420,15 @@ deployments:
 #orderbook-subparser !The subparser for the Orderbook
 
 #plottables-test
+  tranche-io-ratio
   amount
   io-ratio:,
   input-amount: decimal18-mul(amount io-ratio),
-  effective-price: decimal18-inv(io-ratio);
+  effective-price: decimal18-inv(io-ratio),
+  effective-price-twap: decimal18-inv(tranche-io-ratio);
```

Comment:
this isn't always a twap

Reply:
should i just rename a `effective-price-twap`

Reply:
pushed a fix .